web-router.go

```diff
@@ -75,6 +75,7 @@ func registerWebRouter(mux *router.Router, web *webAPIHandlers) {
 	webBrowserRouter.Methods("POST").Path("/webrpc").Handler(webRPC)
 	webBrowserRouter.Methods("PUT").Path("/upload/{bucket}/{object:.+}").HandlerFunc(web.Upload)
 	webBrowserRouter.Methods("GET").Path("/download/{bucket}/{object:.+}").Queries("token", "{token:.*}").HandlerFunc(web.Download)
+	webBrowserRouter.Methods("GET").Path("/heal/{bucket}/{object:.+}").HandlerFunc(web.Heal)
```

Comment:
Is there a reason to keep it like this ? why isn't webrpc sufficient for this use?